website/src/Components/Footer/footer.tsx

```diff
@@ -4,12 +4,16 @@ const Footer = () => {
   return (
     <footer className="flex flex-col items-center bg-top-red">
       <p className="text-white pt-2">Eastlake 2020</p>
-      <a className="text-white" href="mailto:eastlakekey@gmail.com">
-        Issues/Suggestions: eastlakekey@gmail.com
-      </a>
-      <a className="text-white pb-2" href="https://repo.ehs-service.org">
-        About: repo.ehs-service.org
-      </a>
+      <p className="text-white pb-2">
+          This tracker is open source: <a href="https://github.com/eastlakehs/community-service-tracker">
+          https://github.com/eastlakehs/community-service-tracker
+        </a>
```

Comment:
```suggestion
      <p className="text-white pb-2">
          This tracker is open source: <a href="https://github.com/eastlakehs/community-service-tracker">
          contribute here
        </a>
```

I think it's cleaner to not leave in the full url

Reply:
I've also changed the redirect for ```repo.ehs-service.org``` back to this repo so you should be able to use it correctly now if you want to (as always it takes about 24 hours to cache-bust and you might to flush dns locally).

Reply:
Your changes are merged. Do you want me to change it back to `repo.ehs-service.org`?

Reply:
@theTisch21 yeah let's move it back. The DNS is resolving correctly for me now. The original also has the spacing correct (see the problem above that I commented with the footer)